module/VuFind/src/VuFind/OAuth2/Entity/UserEntity.php

```diff
@@ -170,31 +180,31 @@ public function getClaims()
                 case 'full_name':
```

Comment:
Would it make sense to add a second case to support `getFullName` here as well, just for consistency of values? I don't insist on it, it just feels a little weird having two different styles of attributes.

Reply:
There is a kind of consistency in styles, with camelCased ones being user entity methods and snake_cased being either profile fields or the documented special fields. There isn't a `getFullName()` in `UserEntityInterface` but we could of course add it there (and the implementing class)?

Reply:
True, I think you're right that it's better to leave this alone. I think adding getFullName() to `UserEntityInterface` puts us at risk of including business logic in an entity model, and it's probably better to avoid that.